bundles/org.openhab.binding.insteon/src/main/resources/device_features.xml

```diff
@@ -403,6 +403,26 @@
 	<command-handler command="OnOffType">NoOpCommandHandler</command-handler>
 	<poll-handler>NoPollHandler</poll-handler>
 </feature>
+<feature name="WirelessMotionSensorLightLevelAboveThreshold">
+	<message-dispatcher>DefaultDispatcher</message-dispatcher>
+	<message-handler cmd="0x03" group="2">NoOpMsgHandler</message-handler>
+	<message-handler cmd="0x11" group="2">OpenedSleepingContactHandler</message-handler>
+	<message-handler cmd="0x13" group="2">ClosedSleepingContactHandler</message-handler>
+	<message-handler cmd="0x19">NoOpMsgHandler</message-handler>
+	<message-handler cmd="0x2e">NoOpMsgHandler</message-handler>
+	<command-handler command="OnOffType">NoOpCommandHandler</command-handler>
+	<poll-handler>NoPollHandler</poll-handler>
+</feature>
+<feature name="WirelessMotionSensorLowbattery">
```

Comment:
```suggestion
<feature name="WirelessMotionSensorLowBattery">
```

Reply:
No problem. Sorry for the issues. Done now, pushed new version.